src/main/java/roomescape/DAO/ReservationReposiory.java

```diff
@@ -0,0 +1,62 @@
+package roomescape.DAO;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.DTO.ReservationDTO;
+import roomescape.Domain.Reservation;
+import roomescape.Domain.Time;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ReservationReposiory {
+  @Autowired
+  private JdbcTemplate jdbcTemplate;
+  public List<Reservation> getAllReservation() {
+    String sql = "select r.id as reservation_id, r.name, r.date, t.id as time_id, t.time as time_value from reservation as r inner join time as t on r.time_id = t.id";
```

Comment:
java 15부터는 문자열을 여러줄로 작성할 수 있는 multi-line String이 등장했습니다!
쿼리를 조금 더 읽기 쉽게 작성해볼 수도 있을 것 같아요
```suggestion
    String sql = """
    select 
      r.id as reservation_id, 
      r.name, r.date, 
      t.id as time_id, 
      t.time as time_value 
    from reservation as r 
    inner join time as t on r.time_id = t.id
    """;
```

참고자료: https://www.baeldung.com/java-multiline-string

Reply:
df9bf8aa34295feb61e632c6e54c7c938d63fdd9 이런게 있었군요!! 감사합니다